src/views/Routes.tsx

```diff
@@ -272,6 +276,59 @@ export const Routes = () => {
     }
   }, [deletedRoute]);
 
+  const styleNotification = { marginTop: 85 };
+
+  const saveKey = isUpdating ? "Updating" : "Saving";
+  useEffect(() => {
+    if (savedRoute.loading) {
+      message.loading({
+        content: isUpdating ? "Updating..." : "Saving...",
+        key: saveKey,
+        duration: 0,
+        style: styleNotification,
+      });
+    } else if (savedRoute.success) {
+      message.success({
+        content: `Route has been successfully ${
+          isUpdating ? "Updated" : "added"
```

Comment:
```
isUpdating ? "updated" : "added"
```

Reply:
yeah same api is calling on save, so to show different message for update and save

Reply:
I see, that makes sense. This a very minor change but I would suggest using lowercase so `Updated` to `updated`

Reply:
I still see Updated in the capital case in the notification. Was it reverted?

Reply:
done